CHANGELOG.md

```diff
@@ -12,13 +12,19 @@ different versioning scheme, following the Haskell community's
 [package versioning policy](https://wiki.haskell.org/Package_versioning_policy).
 
 ## Unreleased ##
-* `gbc` & compiler library: TBD
+* `gbc` & compiler library: TBD (minor bump needed)
 * IDL core version: TBD
 * IDL comm version: TBD
-* C++ version: TBD
+* C++ version: TBD (major bump needed)
 * C# NuGet version: TBD
 * C# Comm NuGet version: TBD (minor bump needed)
 
+### C++ ###
+* **Breaking change:** Suppression of MSVC name shadowing warnings is no
```

Comment:
Given this, I wonder if we want to bundle this together with more of the warning un-suppression work, Thoughts?